app/controllers/landing_controller.rb

```diff
@@ -1,5 +1,5 @@
 class LandingController < ApplicationController
   def index
-    
+    @restaurants = Restaurant.all
```

Comment:
Again, it does not make sense to keep this controller. `root_path` should, IMO, point to `restaurants_controller#index`